app/models/customer.rb

```diff
@@ -7,9 +7,10 @@ class Customer < ApplicationRecord
 
   def self.top_five
     joins(:transactions)
+    .select('customers.*, count(customers.id) AS customer_count')
     .group(:id)
-    .where('transactions.result = ?', 'success')
-    .order('count(customers.id) desc')
+    .where("transactions.result = 'success'")
+    .order('customer_count desc')
```

Comment:
no more deprication! but can we use less ruby and more active record? im not sure and need more learnin on it